nemo/lightning/_strategy_lib.py

```diff
@@ -516,6 +516,19 @@ def load_model_state_dict(megatron_parallel, checkpoint: Mapping[str, Any], stri
     from megatron.core import parallel_state
     from megatron.core.dist_checkpointing.validation import StrictHandling, parse_strict_flag
 
+    ## convert from StrictHandling to bool for PTL
+    if os.environ.get("MCORE_STRICT_HANDLING") is not None:
```

Comment:
Let's avoid such logic, it will be terrible to debug later on.

What's the reason this can't be passed by argument?

Reply:
I would prefer if it's set in a global var as opposed to a environment variable.